docs/maintain/maintain-endpoints.md

```diff
@@ -62,7 +62,6 @@ maintained by Parity Technologies are listed below:
 | Network | WSS Endpoint                  |
 | ------- | ----------------------------- |
 | Westend | wss://westend-rpc.polkadot.io |
```

Comment:
I guess this is only for Parity endpoints?

Reply:
yeah we do not list community maintained endpoint for the simple fact there are multiple RPCs